### PR TITLE
MNT: Add noarch: python

### PR DESCRIPTION
**Pull request by conda-forge-linter**
As instructed in #1

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
+  noarch: python
   number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml:16]: Should we be bumping the build number?

Comment (isuruf) [reply recipe/meta.yaml:16]: I don't have the permission though

Comment (sodre) [reply recipe/meta.yaml:16]: I will merge it by hand and bump it to build number 1 at the same time.

Comment (jakirkham) [reply recipe/meta.yaml:16]: Sounds good. Maybe we should add an issue to add this for `noarch: python`.

Comment (sodre) [reply recipe/meta.yaml:16]: @isuruf can you take care of adding the issue? I am not sure where to file it exactly.

Comment (jakirkham) [reply recipe/meta.yaml:16]: Sorry for the confusion @sodre. Was mentioning that as something @isuruf or myself should do. FTR it is in issue ( https://github.com/conda-forge/conda-forge-webservices/issues/125 ).

**Pull request merged**